packages/core/src/ssr-query.ts

```diff
@@ -1,20 +1,41 @@
-import {IncomingMessage, OutgoingMessage} from 'http'
+import {IncomingMessage, ServerResponse} from 'http'
+import {log} from '@blitzjs/display'
 import {InferUnaryParam} from './types'
+import {getAllMiddlewareForModule, handleRequestWithMiddleware, MiddlewareResponse} from './middleware'
+import {EnhancedResolverModule} from './rpc'
 
 type QueryFn = (...args: any) => Promise<any>
 
 type SsrQueryContext = {
   req: IncomingMessage
-  res: OutgoingMessage
+  res: ServerResponse
 }
 
 export async function ssrQuery<T extends QueryFn>(
   queryFn: T,
   params: InferUnaryParam<T>,
-  // @ts-ignore unused
-  context: SsrQueryContext,
+  {req, res}: SsrQueryContext,
 ): Promise<ReturnType<T>> {
-  const data = await queryFn(params)
+  const handler = (queryFn as unknown) as EnhancedResolverModule
 
-  return data
+  const middleware = getAllMiddlewareForModule(handler)
+
+  middleware.push(async (_req, res, next) => {
+    const logPrefix = `${handler._meta.name}`
+    console.log('') // New line
```

Comment:
Should be @blitzjs/display log(''), right?

Reply:
fixed